movies_challenge/.gitignore

```diff
@@ -0,0 +1,26 @@
+# See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
+
+# dependencies
+/node_modules
+/src/settings
```

Comment:
It a good practice to never push API keys or secret keys to Github, take a look at dotenv project which is the right way to manage those keys.